osdk/src/cli.rs

```diff
@@ -358,4 +358,10 @@ pub struct CommonArgs {
         global = true
     )]
     pub qemu_args: Vec<String>,
+    #[arg(
+        long = "compression-format",
+        help = "Denote the compression format for kernel self-decompression",
+        global = true
+    )]
+    pub compression_format: Option<String>,
```

Comment:
It is better for `compression_format` to be a enum (`pub compression_format: Option<CompressionFormat>`). You can define the enum in builder and reference it here too.